pkg/acsengine/const.go

```diff
@@ -41,8 +41,8 @@ const (
 )
 
 const (
-	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.5.3"
-	KubernetesDashboardImageName         = "kubernetes-dashboard-amd64:v1.5.1"
+	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.6.0"
```

Comment:
@amanohar Just to confirm, you want something like this in here correct? 
```
KubernetesHyperkubeImageName153         = "hyperkube-amd64:v1.5.3"
KubernetesDashboardImageName153         = "kubernetes-dashboard-amd64:v1.5.3"
KubernetesHyperkubeImageName160         = "hyperkube-amd64:v1.6.0"
KubernetesDashboardImageName160         = "kubernetes-dashboard-amd64:v1.6.0"
```
But how do users pass in which version they want to use?